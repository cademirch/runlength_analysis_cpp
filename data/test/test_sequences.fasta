>test1
ACCAAACCCC
>test2
GGGGTTTGGT
>test3
GGGGTTTGGT
GGGGTTTGGT
GGGGTTTGGT
>test4
ACCAAACCCC
