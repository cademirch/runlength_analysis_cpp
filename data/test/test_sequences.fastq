@a380da2a-082a-4dd0-a5b7-7994d6bb9a8b runid=862ca2bd7743b86d234bcce8a3807b7c75829b36 sampleid=sample_id read=4131 ch=15 start_time=1484-12-31T12:01:43Z
TTCGATATATAAAT
+
#*+(&&')##-/''
@dc83117e-a66b-4ae9-8dc3-5e370467ebf3 runid=862ca2bd7743b86d234bcce8a3807b7c75829b36 sampleid=sample_id read=799 ch=176 start_time=1484-12-31T01:20:52Z
ACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTAC
+
!"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~
