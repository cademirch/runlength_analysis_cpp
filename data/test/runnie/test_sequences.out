# 284eaf32-5758-4df8-90b7-fddb3b4928e7
T	0x1.ada896p+1	0x1.52a314p+0
C	0x1.2727c8p+2	0x1.317bb4p+0
G	0x1.8d1268p+3	0x1.a07718p+0
T	0x1.d258dep+3	0x1.a6863ep+0
A	0x1.f113ccp+1	0x1.eedda4p-4
C	0x1.0259bep+0	0x1.469e84p-3
# f1a9eee6-c30a-47b3-9882-d0f8fe7cf78f
C	0x1.177674p+0	0x1.439ad4p-2
# 6597b991-4bc1-4da3-913d-35482237e725
T	0x1.6dab1ap+3	0x1.9d95dap+0
G	0x1.abcb28p+1	0x1.1cc672p-2
C	0x1.2386a2p+0	0x1.7d1f14p-3
A	0x1.2ad89ap+0	0x1.590b04p-3
T	0x1.2fc432p+0	0x1.4e1b9ep-3
G	0x1.4e95dcp+0	0x1.5eaf7ep-2
C	0x1.14c906p+0	0x1.dc05b8p-3
G	0x1.57710ep+0	0x1.5adacap-3
# 9cc73efa-837f-49c3-93fb-c8273be57181
G	0x1.01a03ap+0	0x1.04924cp-2
A	0x1.07ab0ep+0	0x1.441026p+2
T	0x1.5857bep+0	0x1.77bbbp-2
C	0x1.023bdp+2	0x1.f09dbp-1
A	0x1.53dc28p+0	0x1.efceb8p-3
T	0x1.0be7p+3	0x1.85d4aap+0
G	0x1.e340ecp+0	0x1.b70dd2p-2
C	0x1.0686c6p+0	0x1.4b3e24p-3
T	0x1.2d24f8p+3	0x1.8c0ce2p+0
C	0x1.70f2dcp+0	0x1.9db99p-1
G	0x1.835d8p+0	0x1.7fa1ep-3
T	0x1.18cadcp+3	0x1.11b794p+1
C	0x1.9b63b6p+2	0x1.7ea836p+0
A	0x1.c2a872p+0	0x1.99bdbep-3
G	0x1.21c78cp+0	0x1.5339d8p-3
